schemas/development/openservice.schema.json

```
@@ -0,0 +1,48 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://raw.githubusercontent.com/xarf/schema-discussion/master/schemas/development/openservice.schema.json",
+  "title": "XARF BOTNET",
+  "description": "A format to report open service events.",
+  "allOf": [
+    {
+      "$ref": "xarf_shared.schema.json#/XarfBase"
+    },
+    {
+      "type": "object",
+      "properties": {
+        "Report": {
+          "allOf": [
+            {
+              "$ref": "xarf_shared.schema.json#/IpAndUrlBasedReport"
+            },
+            {
+              "type": "object",
+              "properties": {
+                "ReportClass": {
+                  "type": "string",
+                  "enum": ["Content"]
+                },
+                "ReportType": {
+                  "type": "string",
+                  "enum": ["OpenService"]
+                },
+                "Service": {
```

```suggestion
                "ServiceName": {
```
Maybe `Service` information should be bundled in a shared object?

---

I think that's a good idea if we add more info like version, see below